### PR TITLE
Issue 962: update user mgmt docs

### PR DESCRIPTION
**Pull request by ADPennington**
### Summary of Changes
_Updated screenshots of [user profile](https://github.com/raft-tech/TANF-app/blob/119cf6fdc6995b2a53f90c21bbb18b6657d550af/docs/Technical-Documentation/images/admin_users.png) and [groups](https://github.com/raft-tech/TANF-app/blob/119cf6fdc6995b2a53f90c21bbb18b6657d550af/docs/Technical-Documentation/images/group_list.png) Added info re: UUID since this is captured in user profile screenshot._
_Addresses part of issue #962_

### CODE REVIEW

File: docs/Technical-Documentation/user_role_management.md
Comment (ADPennington) [inline docs/Technical-Documentation/user_role_management.md:148]: @jtwillis92 @abottoms-coder, do the steps below need to be updated? I walked through these a couple of weeks ago and noticed at least line 152 seemed outdated?

**Comment by ADPennington** [comment]
removing this from qasp for now. this is blocked by #930, which will change the user profile (replacing `Active` with `Deactivated`. I will update the screenshot after 930 is merged.

**Comment by ADPennington** [comment]
7/13: holding off until #1071 addressed.

**Comment by ADPennington** [comment]
> @ADPennington Would you like to open an PR for this older branch or should it be deleted?

i think we can keep this PR. since #1124 was merged, I have the info I need to update the screenshots 😄

**Comment by ADPennington** [comment]
@lfrohlich this is ready for your review.

File: docs/Technical-Documentation/user_role_management.md
Comment (ADPennington) [inline docs/Technical-Documentation/user_role_management.md:42]:
```suggestion
- Current groups listed are "System Admin", "OFA Admin" , and "Data Analyst".
```

**Review by lfrohlich — APPROVED** [review]
LGTM!

**Pull request merged**
